### PR TITLE
Update lab 3 to fix small issues

### PR DESCRIPTION
**Pull request by viveksyngh**
Signed-off-by: Vivek Singh <vivekkmr45@yahoo.in>

### CODE REVIEW

File: lab3.md [hunk 1]
```diff
@@ -424,7 +430,7 @@ Official Template: true
 Pull the template down:
 
 ```sh
-$ faas-cli template pull node10-express
+$ faas-cli template pull https://github.com/openfaas-incubator/node10-express-template
```
Comment (alexellis) [inline lab3.md]: Please change to `faas-cli template store pull`

File: lab3.md [hunk 2]
```diff
@@ -223,6 +223,12 @@ $ faas-cli build -f ./astronaut-finder.yml
 
 > Tip: Try renaming astronaut-finder.yml to `stack.yml` and calling just `faas-cli build`. `stack.yml` is the default file-name for the CLI.
 
+
+Push the function:
+```
+$ faas-cli push -f ./astronaut-finder.yml
```
Comment (alexellis) [inline lab3.md]: :+1:

File: lab3.md (hunk 1 above)
Comment (viveksyngh) [reply lab3.md]: Ah ! Okay. I will update that.

Comment (viveksyngh) [reply lab3.md]: Updated it.

**Comment by alexellis** [comment]
Thank you Vivek. Please keep the PRs coming as needed

**Pull request merged**